control-plane/api/v1alpha1/routeauthfilter_types.go

```diff
@@ -26,8 +26,8 @@ type RouteAuthFilter struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   RouteAuthFilterSpec `json:"spec,omitempty"`
-	Status `json:"status,omitempty"`
+	Spec   RouteAuthFilterSpec   `json:"spec,omitempty"`
+	Status RouteAuthFilterStatus `json:"status,omitempty"`
```

Comment:
```suggestion
	Status            `json:"status,omitempty"`
```

Is how I see it defined for the other CRDs

Reply:
that's true, but for the gateway CRDs we include the `generation` in the condition which is not part of our `Status` type (and this follows in line with how the other gateway types handle status in the sig https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1beta1/gateway_types.go#L45, https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1beta1/httproute_types.go#L43